west.yml

```diff
@@ -192,7 +192,7 @@ manifest:
     - name: openthread
       repo-path: sdk-openthread
       path: modules/lib/openthread
-      revision: 2060039bbfec52ac00f22a7cc3213c2ab1df138e
+      revision: 0a79c30a202cf0c96d2ce5e37ee01a0546ed40c5
```

Comment:
OT is already at `a4f1c7afa729733d4038c21e1750e469fbbb9c8e` which is ahead of this commit. Is going backwards in the history really intended?

Reply:
No this is my mistake due to not triggered CI previously I will re trigger the upmerge job to fix it.